docs/src/building.md

```diff
@@ -71,3 +71,13 @@ $ nix build .#robotnixConfigurations.dailydriver.img
 ```
 This is assuming the `flake.nix` is in the current directory, the desired configuration is named `dailydriver`, and will produce the `img` output.
 Other robotnix outputs are available using a similar command.
+
+## External nix store use
+
+If your main volume is of a smaller capacity and you wish to use an external disk to hold the nix store, then use the following example 
+
+```shell
+$ nix build -L --store "<External Drive or NFS>/Android/store" --option extra-sandbox-paths "/keys=$(pwd)/keys /var/cache/ccache?" -j$(nproc) --cores $(nproc) "$@"
```

Comment:
I think a lot of stuff in this command line is irrelevant to the problem and just amounts to confusion.

Reply:
> I think a lot of stuff in this command line is irrelevant to the problem and just amounts to confusion.

I agree, I was trying to consolidate the snippets of information from the docs into what could be a final command line, without creating additional PRs. How that is presented I don't mind, nor do I mind just something like this remaining...

`nix build -L --store "<External Drive or NFS>/Android/store"`